app/directives.js

```diff
@@ -4,71 +4,12 @@ airTrafficControlApp.directive('aircraftDetails', function(){
       templateUrl:'directives/aircraftDetails.html',
       scope:{
          aircraftObject: "="

```

Comment:
Please remove this; accomplish this in a different way; this is a 'weird' implementation, and we do not want these.